contracts/Receivable.sol

```diff
@@ -24,28 +24,34 @@ contract Receivable is
     ERC721EnumerableUpgradeable,
     ERC721BurnableUpgradeable,
     UUPSUpgradeable,
-    Governable,
     IReceivable
 {
+    address public governor;
```

Comment:
I recommend placing the `governor` address in the `receivableStorage` contract to prevent potential storage collision issues when new implementations inherit the storage layout.